### DISCUSSION TITLE
Amethyst Initial Version

### DISCUSSION THREAD

**Discussion by rckarns8** [rckarns8/amethyst#4] [General]
This version includes the following functionality:

| Program      | Function                          | Status      |
|--------------|-----------------------------------|-------------|
| fastqc       | read QC                           | complete    |
| multiqc      | read QC                           | complete    |
| multitrim    | read trimming                     | complete    |
| megahit      | assembly                          | complete    |
| bbnorm       | normalization                     | ON HOLD     |
| bowtie2      | assembly coverage analysis        | complete    |
| prodigal     | gene prediction                   | complete    |
| prokka       | gene annotation                   | complete    |
| sourmash     | diversity analysis- assembly based| complete    |
| maxbin2      | binning                           | complete    |
| checkm       | bin/MAG quality                   | complete    |
| dRep         | dereplication of bins/MAGs        | complete    |
| GTDB-Tk      | taxonomic assignment of bins/MAGs | complete    |
| nonpareil    | read-based coverage analysis      | backlog     |
| sourmash     | diversity analysis- read based    | backlog     |
| humann       | functional analysis               | backlog     |

<hr /><em>This discussion was created from the release <a href='https://github.com/rckarns8/amethyst/releases/tag/'>Amethyst Initial Version</a>.</em>

**Comment by lukenoaa** [comment]
Is this an older version of discussion #5? Maybe we can archive this one?
